scripts/buildAndRunTests.sh

```diff
@@ -1,33 +1,82 @@
 #!/bin/bash
 
+pwd=`pwd`
+
+function usage () {
+  echo "-h : display help & exit"
+  echo "-skipg3log : skip building g3log"
+  echo "-skiptests : skip building & executing tests"
+}
+
+install_g3log=true
+do_tests=true
+while [ "$1" != "" ]; do
+  case $1 in
+    -skipg3log ) install_g3log=false
+                 ;;
+    -skiptests ) do_tests=false
+                 ;;
+    -h ) usage
+         exit
+         ;;
+    * ) usage
+        exit 1
+  esac
+  shift
+done
+
 set -ev
 set -x
 
-pwd=`pwd`
-mkdir g3log-install && cd g3log-install
-git clone https://github.com/KjellKod/g3log.git -b master
-cd g3log
-unzip -o 3rdParty/gtest/gtest-1.7.0.zip -d 3rdParty/gtest
-mkdir -p  build_travis
-cd build_travis
-cmake  -DADD_G3LOG_UNIT_TEST=ON ..
-make -j && sudo make install
+curdir=${PWD##*/}
+if [ $curdir != "g3sinks" ]; then
+  echo "not in the right directory. Should be in g3sinks. abord."
+  exit 1
+fi
+
+
+function install_g3log () {
+  mkdir g3log-install && cd g3log-install
+  git clone https://github.com/KjellKod/g3log.git -b master
+  cd g3log
+  unzip -o 3rdParty/gtest/gtest-1.7.0.zip -d 3rdParty/gtest
+  mkdir -p  build_travis
+  cd build_travis
+  cmake  -DADD_G3LOG_UNIT_TEST=ON ..
+  make -j && sudo make install
+  }
 
+if [ "$install_g3log" == true ]; then
+  install_g3log
+else
+  echo "SKIPPING g3log build."
+fi
 
 # Build g3sinks
 cd $pwd
-cd 3rdparty && unzip gtest-1.7.0.zip
-cd $pwd
+mkdir build 
 
-#logrotate
-echo "BUILD LOGROTATE SINK AND RUN TESTS"
-cd logrotate && mkdir build && cd build && cmake -DADD_LOGROTATE_UNIT_TEST=ON ..
-make -j && ./UnitTestRunner
-echo "FINISHED LOGROTATE BUILD AND TESTS"
+if [ $do_tests == true ]; then
```

Comment:
Why is this needed?  If one runs the `buildAndrunTests.sh` scripts I think it's assumed that you will build the tests, right?

Reply:
Actually I found the script convenient to debug the cmake files with different options (shared or static), it's also useful to check for syntax errors etc... So I found it convenient to have an option to disable the tests, and also disable installing g3log.
This is by no means a required change, only a small upgrade to the script that I found convenient, and I though it could come in handy to others too.

Reply:
I've removed the option to skip the tests, I wait for this to be merged before I submit another PR for issue #78

Reply:
(Finally I didn't wait for this to be merged to submit the other PR, but the two PRs are really independent, so currently I've two PRs open in g3sinks and one open in g3log )